.gitlab-ci.yml

```diff
@@ -42,6 +42,30 @@ test:
     paths:
       - coverage.lcov
 
+backwards-compat:
+  stage: test
+  image: debian:10
+  before_script:
+    - apt update && apt install -yyq g++ git make lcov pkg-config liblmdb++-dev libboost-dev libboost-log-dev libboost-regex-dev libssl-dev libarchive-dev libdbus-1-dev curl dbus
```

Comment:
Can you add `stunnel4 tinyproxy-bin` right away? Then we won't get a mismerge with #1433.

Reply:
This job is meant to only build, not to run tests, so it shouldn't be necessary.

Reply:
Doh! Of course. As pointed out by me also ten seconds later! 😄